src/components.rs

```diff
@@ -180,3 +174,11 @@ pub struct ControllerInput {
     /// Will be reset to 0 after being applied.
     pub custom_torque: Vec3,
 }
+
+/// Entities that should be considered as part of the controlled character.
+#[derive(Deref, DerefMut, Component, Default, Reflect)]
+#[reflect(Component)]
+pub struct RelatedEntities {
```

Comment:
I think this is not quite the right way to go about this. Firstly, naming this "RelatedEntities" instead of "ExcludeFromGround" obscures the actual meaning of what this does. Secondly, adding it as a component instead of an option directly on the ControllerSettings limits discoverability and makes it less clear what it's related to.